sdk/eventhub/Azure.Messaging.EventHubs/tests/Primitives/EventProcessorTests.PartitionProcessing.cs

```diff
@@ -119,6 +119,81 @@ public async Task ProcessEventBatchAsyncDoesNotInvokeTheHandlerWhenEmptyBatchesS
                 .Verify("OnProcessingEventBatchAsync", Times.Never(), EmptyBatch, partition, cancellationSource.Token);
         }
 
+        /// <summary>
+        ///   Verifies functionality of the <see cref="EventProcessor{TPartition}.ProcessEventBatchAsync" />
+        ///   method.
+        /// </summary>
+        ///
+        [Test]
+        public async Task ProcessEventBatchAsyncLogsWhenBatchesAreDispatched()
```

Comment:
Is it worth using the TestEventListener (whether in live tests or unit tests) to verify that the events are being logged as expected?

Reply:
Currently all of the verification that we're doing is unit tests only.  I've got an issue open to take a look at integrating `TestEventListener` for all of our logs across the project as part of Live coverage.  I think there's value in doing it as a whole, but not so much for just covering this change.